trillian/ctfe/handlers.go

```diff
@@ -183,6 +185,13 @@ type CertValidationOpts struct {
 	trustedRoots *PEMCertPool
 	// rejectExpired indicates whether certificate validity period should be used during chain verification
 	rejectExpired bool
+	// notAfterStart is the earliest notAfter date which will be accepted.
```

Comment:
nil means no checking, Zero time would be a cut off of whatever time that works out to (epoch, presumably?)